classes/options-general.php

```diff
@@ -328,51 +328,51 @@ function print_page() {
-																	value="postalcode town, province" <?php selected( $address_format, 'postalcode town, province' ); ?>><?php echo '[' . __( 'Zip/Postal Code', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'City/Town', 'simplemap' ) . '], [' . __( 'State/Province', 'simplemap' ) . ']'; ?></option>
+																	value="postalcode town, province" <?php selected( $address_format, 'postalcode town, province' ); ?>><?php echo '[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'City/Town', 'SimpleMap' ) . '], [' . __( 'State/Province', 'SimpleMap' ) . ']'; ?></option>
 
 																<option
-																	value="postalcode town" <?php selected( $address_format, 'postalcode town' ); ?>><?php echo '[' . __( 'Zip/Postal Code', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'City/Town', 'simplemap' ) . ']'; ?></option>
+																	value="postalcode town" <?php selected( $address_format, 'postalcode town' ); ?>><?php echo '[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'City/Town', 'SimpleMap' ) . ']'; ?></option>
 
 																<option
-																	value="town postalcode" <?php selected( $address_format, 'town postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'simplemap' ) . ']'; ?></option>
+																	value="town postalcode" <?php selected( $address_format, 'town postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']'; ?></option>
 															</select>
 															<span class="hidden"
-															      id="order_1"><br/><?php _e( 'Example', 'simplemap' ); ?>
+															      id="order_1"><br/><?php _e( 'Example', 'SimpleMap' ); ?>
 																: Minneapolis, MN 55403</span>
 															<span class="hidden"
-															      id="order_2"><br/><?php _e( 'Example', 'simplemap' ); ?>
+															      id="order_2"><br/><?php _e( 'Example', 'SimpleMap' ); ?>
 																: Minneapolis MN 55403</span>
 															<span class="hidden"
-															      id="order_3"><br/><?php _e( 'Example', 'simplemap' ); ?>
+															      id="order_3"><br/><?php _e( 'Example', 'SimpleMap' ); ?>
 																: S&atilde;o Paulo - SP 85070</span>
 															<span class="hidden"
-															      id="order_4"><br/><?php _e( 'Example', 'simplemap' ); ?>
+															      id="order_4"><br/><?php _e( 'Example', 'SimpleMap' ); ?>
 																: 85070 S&atilde;o Paulo - SP</span>
 															<span class="hidden"
-															      id="order_5"><br/><?php _e( 'Example', 'simplemap' ); ?>
+															      id="order_5"><br/><?php _e( 'Example', 'SimpleMap' ); ?>
 																: 46800 Puerto Vallarta, JAL</span>
 															<span class="hidden"
-															      id="order_6"><br/><?php _e( 'Example', 'simplemap' ); ?>
+															      id="order_6"><br/><?php _e( 'Example', 'SimpleMap' ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 122 characters

<!-- Comment by [SideCI](https://sideci.com) -->